version.sbt

```diff
@@ -0,0 +1 @@
+ThisBuild / version := "3.1.1-SNAPSHOT"
```

Comment:
Is keeping the version in a separate file more convenient? With my gt-contrib and gt-server experiences I didn't notice any real wins ):

Reply:
Opening the discussion with this. Felt like I should pull this out of `Version.scala` which always stumps me. I'd be also OK with putting this version into `build.sbt` with other build settings.

Reply:
Let’s try with a version in a file; but I’m just worrying that it would be just an extra file and the story would be the same as a Versions class in a separate file.

Reply:
Let’s try with a version in a file; but I’m just worrying that it would be just an extra file and the story would be the same as a Versions class in a separate file.